stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/UserDtos.kt

```diff
@@ -84,3 +84,10 @@ internal data class PartialUpdateUserDto(
     val set: Map<String, Any>,
     val unset: List<String>,
 )
+
+@JsonClass(generateAdapter = true)
+internal data class DownstreamUserBlockDto(
+    val blocked_by_user_id: String,
+    val blocked_user_id: String,
+    val created_at: Date,
```

Comment:
Can we make these fields `nullable` just in case something gets broken on backend side? :)

Reply:
That way we are hiding a potential error and moving the point of failure deeper into the SDK.